### PR TITLE
Fix php examples

### PR DESCRIPTION
**Pull request by faiqibrahim**
Fixes PHP examples 

1. Fix the JWT fetcher method for the Rest
2. Fix the JWT fetcher for gRPC
3. Fixes the response in gRPC examples of DeleteDocument, as that does not contain response code

### CODE REVIEW

File: language-examples/php/grpc/deleteDocument.php [hunk 1]
```diff
@@ -55,12 +55,9 @@
...
 
-if ($result->getStatus()->getCode() === 0) {
+if ($status->code === 0) {
     echo 'SUCCESS: Document deleted';
 } else {
-    echo 'ERROR: Code: ' .
-        $result->getStatus()->getCode() .
-        ' Detail: ' .
-        $result->getStatus()->getStatusDetail();
+    echo 'ERROR: Could not delete document, Status Code: '. $status->code;
```
Comment (aamirbutt) [inline language-examples/php/grpc/deleteDocument.php:61]: Why remove the status detail from the message?

File: language-examples/php/rest/getJwtToken.php [hunk 2]
```diff
@@ -10,18 +10,24 @@
...
         'grant_type' => 'client_credentials',
         'client_id' => $client_id,
+        'client_secret' => $client_secret
     ];
 
     $ch = curl_init();
     curl_setopt($ch, CURLOPT_URL, $url);
     curl_setopt($ch, CURLOPT_POST, true);
-    curl_setopt($ch, CURLOPT_HTTPHEADER, ['Authorization: Basic ' . $encoded]);
+    curl_setopt($ch, CURLOPT_HTTPHEADER, ['Content-Type: application/x-www-form-urlencoded']);
```
Comment (aamirbutt) [inline language-examples/php/rest/getJwtToken.php:30]: I am curious. Why change to this type of auth? Was the bearer token auth not working?

File: language-examples/php/grpc/deleteDocument.php (hunk 1 above)
Comment (aamirbutt) [reply language-examples/php/grpc/deleteDocument.php:61]: Nevermind, I just noticed that there is no Status in DeleteDocumentResponse.

File: language-examples/php/rest/getJwtToken.php (hunk 2 above)
Comment (faiqibrahim) [reply language-examples/php/rest/getJwtToken.php:30]: I just synced the header with other examples in NodeJs, hence adding 'Content-Type'. But it was not working with Basic Auth.

**Review by aamirbutt — APPROVED** [review]

**Pull request merged**